Sources/CoreMetrics/Metrics.swift

```diff
@@ -363,6 +365,8 @@ public enum MetricsSystem {
     fileprivate static let lock = ReadWriteLock()
     fileprivate static var _factory: MetricsFactory = NOOPMetricsHandler.instance
     fileprivate static var initialized = false
+    fileprivate static var systemMetricsHandler: SystemMetricsHandler?
+    fileprivate static var systemMetricsInitialized = false
```

Comment:
do we need two flags? isn't systemMetricsHandler being nil a good enough indication?

Reply:
Good point. Will remove the flag